packages/headless/src/state/product-recommendations-app-state.ts

```diff
@@ -2,13 +2,13 @@ import {
   ConfigurationSection,
```

Comment:
You should be able to remove the file entirely

Reply:
Donne in 682d339